source/_integrations/solax.markdown

```diff
@@ -39,6 +39,22 @@ template:
 
 {% endraw %}
 
+### Configuring the Energy Dashboard
+
+There are generally 3 sensors from your inverter that you need to configure in the energy dashboard:
+
+- the total consumption sensor (in kWh) for the "Grid Consumption"
+- the total feed-in sensor (in kWh) for the "Return to grid", i.e. the solar panel energy that you do not consume and return to the grid instead
```

Comment:
Isn't this country depended? I know this ain't going to work in the Netherlands for example.

Reply:
Depends on the setup. There are many ways this can be hooked up. For example, inverters don't have to be connected directly to the grid.